dbt_subprojects/dex/models/_log_decoded_trades/arbitrum/decoded-in-flight/uniswap-v3/uniswap_v3_fork_mapping_arbitrum.sql

```diff
@@ -0,0 +1,19 @@
+-- this should probably live somewhere else, just for testing purposes for now
+
+{{ config(
+    schema = 'dex_mass_decoding_arbitrum',
+    alias = 'uniswap_v3_fork_mapping',
+    tags = ['static'],
+    unique_key = ['factory_address'])
+}}
+
+SELECT factory_address, project_name
+FROM
+(VALUES
+    (0x5c69bee701ef814a2b6a3edd4b1652cb9cc5aa6f, 'uniswap_v2')
+    , (0xc0aee478e3658e2610c5f7a4a2e1777ce9e4f2ac, 'sushi')
```

Comment:
Not very accurate at the moment,
we may need to use some scripts/tools to generate mapping

Reply:
```suggestion
    (0x1F98431c8aD98523631AE4a59f267346ea31F984, 'uniswap_v3')
```